assets/stylesheets/style.scss

```diff
@@ -43,10 +43,16 @@
 			text-shadow: none !important;
 		}
 		//wp-admin styles override the disabled button with !important, so need to do the same here
+		
 		&.is-busy {
-			background-size: 120px 100% !important;
-			background-image: linear-gradient( -45deg, var( --color-primary ) 28%, var( --color-primary-400 ) 28%, var( --color-primary-400 ) 72%, var( --color-primary ) 72%) !important;
-			border-color: #0081a9 !important;
+			background-image: linear-gradient(
+				-45deg,
+				var( --color-accent ) 28%,
+				var( --color-accent-600 ) 28%,
+				var( --color-accent-600 ) 72%,
+				var( --color-accent ) 72%
+			);
```

Comment:
The previous code had some `!important`, due to L45.

I checked out this branch, and with the changes as they are, the buttons look gray:
![2020-08-12 14 19 05](https://user-images.githubusercontent.com/273592/90058151-f87f5e00-dca6-11ea-8194-0fe3a2c65d5c.gif)
![2020-08-12 14 18 30](https://user-images.githubusercontent.com/273592/90058155-f917f480-dca6-11ea-8434-54005fa584cb.gif)


But as soon as the `!important` is re-introduced, then I see the pink color:
![2020-08-12 14 17 44](https://user-images.githubusercontent.com/273592/90058198-07fea700-dca7-11ea-9c53-8e191c05b46c.gif)

I tested this in Chrome & Firefox.

Should the `!important` be re-added?

Reply:
Thanks for this catch! I've added in the `!important`. I noticed the busy animation is broken by a disabled background attribute so created a new issue #2138 for that.